### PR TITLE
New parameters for SmbShare

### PR DESCRIPTION
**Pull request by Xelu86**
# PR Summary

New parameters have been added to SmbShare for WS25. Updated related cmdlets to conform to style guide.

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: docset/winserver2025-ps/smbshare/Get-SmbShare.md [hunk 1]
```diff
@@ -16,28 +16,33 @@ Retrieves the SMB shares on the computer.
...
+being displayed by the computer.
 
 ## EXAMPLES
 
 ### Example 1: Get SMB shares on a local computer
-```
-PS C:\>Get-SMBShare
+
+```powershell
+PS C:\\>Get-SMBShare
```
Comment (nedpyle) [inline docset/winserver2025-ps/smbshare/Get-SmbShare.md:40]: All of these double \\ are wrong (unless this is some escape character for markup?)

Get-smbshare

Name   ScopeName Path       Description
----   --------- ----       -----------
ADMIN$ *         C:\WINDOWS Remote Admin
C$     *         C:\        Default share
IPC$   *                    Remote IPC

File: docset/winserver2025-ps/smbshare/Get-SmbShare.md [hunk 2]
```diff
@@ -381,8 +487,9 @@ Accept wildcard characters: False
 ```
 
 ### -SmbInstance
-Specifies the input to this cmdlet.
-You can use this parameter, or you can pipe the input to this cmdlet.
+
+Specifies the input to this cmdlet. You can use this parameter, or you can pipe the input to this
```
Comment (nedpyle) [inline docset/winserver2025-ps/smbshare/Get-SmbShare.md:491]: Specifies the SMB share instance type. The default instance is used for operations like a client opening a file on a share on the server. The CSV instance is used by Windows Failover Clusters. The SBL instance is used by Storage Spaces Direct. The SR instance is used by Storage Replica.

File: docset/winserver2025-ps/smbshare/Get-SmbShare.md [hunk 3]
```diff
@@ -304,11 +335,13 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -LeasingMode
-Specifies SMB leasing and oplock behaviors.
+### -IsolatedTransport
```
Comment (nedpyle) [inline docset/winserver2025-ps/smbshare/Get-SmbShare.md:338]: I need to look at this code a bit more for the help, standby on this one.

**Review by nedpyle — CHANGES_REQUESTED** [review]
Some changes needed, plus one I need to get back to you on.

File: docset/winserver2025-ps/smbshare/Get-SmbShare.md (hunk 3 above)
Comment (robinharwood) [reply docset/winserver2025-ps/smbshare/Get-SmbShare.md:338]: @nedpyle - as discussed

Comment (nedpyle) [reply docset/winserver2025-ps/smbshare/Get-SmbShare.md:338]: Help:

"Treat this share as if it is a separate file server instance. Clients will create a new set of connections to that share." 

or to this effect. This was built for Azure but we can't hide it, there's no good reason for customers to use it and we don't recommend they use it.

**Pull request merged**